### PR TITLE
Resolves #2691, Approved Progs banked months tikler enhancement

### LINKED ISSUE #2691: Banked Months TIKLer enhancement for Approved Progs
Idea: with the maxis improvements for banked months eliminating monthly PEPR dails, setting a reminder to workers to review status and update WREG each banked month is needed.  I propose enhancing approved progs to create a TIKL for each banked month noted as approved.

### PR DESCRIPTION
**Pull request by courtrightd**
Blip: The banked months tracking spreadsheet has been removed, and the script will now automatically generate a TIKL for each future banked month approved telling the worker to update WREG tracking and approve results.

### CODE REVIEW

File: Script Files/NOTES/NOTES - APPROVED PROGRAMS.vbs [hunk 1]
```diff
@@ -238,170 +238,92 @@ IF SNAP_banked_mo_check = checked THEN
...
-						Next
+				month_to_tikl = Trim(month_to_tikl)
+				IF len(month_to_tikl) = 4 THEN month_to_tikl = "0" & month_to_tikl
+				tikl_month_mm = left(month_to_tikl,2)
+				tikl_month_yy = right(month_to_tikl,2)
+				tikl_date = tikl_month_mm & "/01/20" & tikl_month_yy
+				IF cdate(tikl_date) > date THEN
+					navigate_to_MAXIS_screen "DAIL", "WRIT"
+
+					MsgBox tikl_date'We can only enter TIKL's for tomorrow or later
```
Comment (CDPotter) [inline Script Files/NOTES/NOTES - APPROVED PROGRAMS.vbs]: Was this from testing?

Comment (CDPotter) [inline Script Files/NOTES/NOTES - APPROVED PROGRAMS.vbs]: Also please see fake case 246333. I ran this twice and it seems some blank items are being added to the array.

Comment (CDPotter) [inline Script Files/NOTES/NOTES - APPROVED PROGRAMS.vbs]: I was running it for 12/16 approval, snap, banked months, autofill from 12/16, banked months for memb 1 starting 1/17 (3 months and memb 3 starting 12/16 for 1 month.

As well as:

12/16 approval, snap, banked months, autofill from 12/16, banked months for memb 1 starting 12/16 for 3 months and memb 3 starting 11/16 for 2 month.

**Comment by courtrightd** [comment]
Changes have been committed, should all be working now.

**Comment by CDPotter** [comment]
Now I'm getting a script error 
image

**Comment by courtrightd** [comment]
That was a major copy-paste error.   Stand by .

**Pull request merged**